widgets/toggle/toggle.js

```diff
@@ -62,8 +66,41 @@ function toggle({
     getConfiguration: () => ({
       facets: [facetName]
     }),
+    init: (state, helper) => {
+      if (userValues.off === undefined) {
+        return;
+      }
+      // Add filtering on the 'off' value if set
+      let isRefined = state.isFacetRefined(facetName, userValues.on);
+      if (!isRefined) {
+        helper.addFacetRefinement(facetName, userValues.off);
+      }
+    },
+    toggleRefinement: (helper, isRefined) => {
+      let on = userValues.on;
+      let off = userValues.off;
+      let hasAnOffValue = (off !== undefined);
+
+      // Checking
+      if (!isRefined) {
+        if (hasAnOffValue) {
```

Comment:
I think it's a bit ugly to have this kind of if, when you can provide alternative implementation. But I don't know if the benefit would be huge here, compared to the troubles.